src/CommunityToolkit.Maui.Core/Essentials/FileSaver/FileSaver.shared.cs

```diff
@@ -11,13 +13,23 @@ public static class FileSaver
 	public static IFileSaver Default => defaultImplementation.Value;
 
 	/// <inheritdoc cref="IFileSaver.SaveAsync(string, string, Stream, CancellationToken)"/>
+	[SupportedOSPlatform("Android26.0")]
+	[SupportedOSPlatform("iOS14.0")]
+	[SupportedOSPlatform("MacCatalyst14.0")]
+	[SupportedOSPlatform("Tizen")]
+	[SupportedOSPlatform("Windows")]
 	public static Task<FileSaverResult> SaveAsync(string initialPath, string fileName, Stream stream, CancellationToken cancellationToken) =>
 		Default.SaveAsync(initialPath, fileName, stream, cancellationToken);
 
 	/// <inheritdoc cref="IFileSaver.SaveAsync(string, Stream, CancellationToken)"/>
+	[SupportedOSPlatform("Android")]
+	[SupportedOSPlatform("iOS14.0")]
+	[SupportedOSPlatform("MacCatalyst14.0")]
+	[SupportedOSPlatform("Tizen")]
+	[SupportedOSPlatform("Windows")]
```

Comment:
I'm not aware of any problems with this feature on iOS 13 and lower? Does this feature not work on iOS 13, for example? 

If there is a known-issue on iOS 13, let's open an Issue and comment here with the link. 

If not, we can remove all of these attributes.

```suggestion
```